packages/frint-store/src/createStore.spec.js

```diff
@@ -32,8 +32,6 @@ describe('frint-store › createStore', function () {
         expect(state).to.deep.equal({
           ok: true,
         });
-
-        done();
       });
 
     subscription.unsubscribe();
```

Comment:
wouldn't it be cleaner to do something like:
```
it('returns initial state upon subscription', function () {
      const Store = createStore();
      const store = new Store({
        enableLogger: false,
       initialState: {
         ok: true,
       }
     });
 
     const subscription = store.getState$()
       .subscribe(function (state) {
          expect(state).to.deep.equal({
            ok: true,
          });
         return  subscription.unsubscribe();
        });
       return subscription;
});
```

If you return an async method, then `done` is optional.

Reply:
It's currently following the same pattern as rest of the tests do. @fahad19 has plans to move from `chai` to `jest` or something similar to improve asserting reactive flows. This means that the tests will be reworked soon, so there's no value in perfecting them at this point.